app/views/admin/world_locations/show.html.erb

```diff
@@ -7,7 +7,7 @@
       <span class="name"><%= @world_location.name %></span>
       &ldquo;<span class="title"><%= @world_location.title %></span>&rdquo;
     </h1>
-    <p><%= view_on_website_link_for @world_location %></p>
```

Comment:
I think the problem is that we're after the URL for the WorldLocationNews page (rather than the WorldLocation), which isn't really backed up by a model. Making the `view_on_website_link_for` method support this specific behaviour might be tricky...